src/sql/run.py

```diff
@@ -154,6 +154,23 @@ def _repr_html_(self):
         else:
             return None
 
+    def _repr_text(self):
```

Comment:
We need to run this function (from `__repr__`) or we can just override `__repr__`

Reply:
Update it now